src/program/program.spec.ts

```diff
@@ -370,4 +386,37 @@ describe("Collection", () => {
 	test("should populate previous and next based on module ID", function () {
 		expect(true).toBe(true);
 	});
+	test("should previous and next based on module ID", function () {
+		expect(false).toBe(true);
+	});
+
+	describe("Creates", () => {
+		describe("Query.createcontent()", () => {
+			test("should create a new module", async () => {
+				testingModuleCreateContentArgs = {
+					type: ContentType.PDF,
+					link: "test",
+					parent: fakelessonID,
+					primary: false
+				};
+				createNewContent = await resolver.createContent(
+					testingModuleCreateContentArgs
+				);
+				if (createModule instanceof Error) throw new Error();
```

Comment:
What is this if statement checking? Why isn't a message added to the Error instance?